### PR TITLE
Release notes for 0.33

### PR DESCRIPTION
**Pull request by trbromley**
Updates to the release notes for 0.33.

Replaces the older PR: https://github.com/PennyLaneAI/pennylane/pull/4647

### CODE REVIEW

File: doc/releases/changelog-0.33.0.md [hunk 1]
```diff
@@ -65,239 +93,272 @@
...
+  ```
 
-* Operator transforms `qml.matrix`, `qml.eigvals`, `qml.generator`, and `qml.transforms.to_zx` are updated
-  to the new transform program system.
-  [(#4573)](https://github.com/PennyLaneAI/pennylane/pull/4573)
+  <img src="https://docs.pennylane.ai/en/stable/_images/cosine_window.png" width=50%/>
 
-* Transforms can be applied on devices following the new device API.
- [(#4667)](https://github.com/PennyLaneAI/pennylane/pull/4667)
+* Controlled gate sequences raised to decreasing powers, a sub-block in quantum phase estimation, can now be created with the new
```
Comment (isaacdevlugt) [inline doc/releases/changelog-0.33.0.md:154]: @KetpuntoG thoughts here?

File: doc/releases/changelog-0.33.0.md [hunk 2]
```diff
@@ -65,239 +93,272 @@
...
-  [#4523](https://github.com/PennyLaneAI/pennylane/pull/4523)
-  [#4524](https://github.com/PennyLaneAI/pennylane/pull/4524)
-  [#4626](https://github.com/PennyLaneAI/pennylane/pull/4626)
-  [#4634](https://github.com/PennyLaneAI/pennylane/pull/4634)
+  jitted_execute = jax.jit(qml.execute, static_argnames=execute_kwargs.keys())
 
-* `MeasurementProcess` and `QuantumScript` objects are now registered as jax pytrees.
-  [(#4607)](https://github.com/PennyLaneAI/pennylane/pull/4607)
-  [(#4608)](https://github.com/PennyLaneAI/pennylane/pull/4608)
+  jitted_execute((tape0, ), **execute_kwargs)
```
Comment (isaacdevlugt) [inline doc/releases/changelog-0.33.0.md:304]: missing output?

File: doc/releases/changelog-0.33.0.md [hunk 3]
```diff
@@ -65,173 +93,238 @@
...
+  ```
+  
+  ```pycon
+  >>> print(draw)
+  0: ──RX──H──┤  <Z>
+  1: ──H───RY─┤
+  2: ──RZ─────┤
   ```
+  
+  Stay tuned for more integration of Catalyst into PennyLane!
```
Comment (trbromley) [inline doc/releases/changelog-0.33.0.md:283]:
```suggestion
```

File: doc/releases/changelog-0.33.0.md [hunk 4]
```diff
@@ -578,11 +673,14 @@ This release contains contributions from (in alphabetical order):
 
 Guillermo Alonso,
```
Comment (isaacdevlugt) [inline doc/releases/changelog-0.33.0.md:677]: Entry on line 635 is missing a PR number:

```
* Fixes `LocalHilbertSchmidt.compute_decomposition` so the template can be used in a qnode.
```

Comment (isaacdevlugt) [reply doc/releases/changelog-0.33.0.md:677]: Entry on line 646 is missing a PR number:

```
* `convert_to_numpy_parameters` now uses `qml.ops.functions.bind_new_parameters`. This reinitializes the operation and
  makes sure everything references the new numpy parameters.
```

File: doc/releases/changelog-0.33.0.md [hunk 5]
```diff
@@ -479,97 +580,94 @@
...
-* Minor documentation improvement for the use of the `pennylane` style of `qml.drawer` and the
-  `pennylane.drawer.plot` style of `matplotlib.pyplot`. The use of the default font was clarified.
+* A documentation improvement for the use of the `pennylane` style of `qml.drawer` and the
+  `pennylane.drawer.plot` style of `matplotlib.pyplot` has been made by clarifying the use of the default font.
   [(#4690)](https://github.com/PennyLaneAI/pennylane/pull/4690)
 
 <h3>Bug fixes 🐛</h3>
 
-* Fixes `LocalHilbertSchmidt.compute_decomposition` so the template can be used in a qnode.
+* Fixed `LocalHilbertSchmidt.compute_decomposition` so that the template can be used in a QNode.
```
Comment (isaacdevlugt) [inline doc/releases/changelog-0.33.0.md:638]: missing PR number

File: doc/releases/changelog-0.33.0.md [hunk 6]
```diff
@@ -479,97 +580,94 @@
...
 
-* Fixed issue where `__copy__` method of the `qml.Select()` operator attempted to access un-initialized data.
+* Fixed an issue where the `__copy__` method of the `qml.Select()` operator attempted to access un-initialized data.
 [(#4551)](https://github.com/PennyLaneAI/pennylane/pull/4551)
 
-* Fix `skip_first` option in `expand_tape_state_prep`.
+* Fixed the `skip_first` option in `expand_tape_state_prep`.
   [(#4564)](https://github.com/PennyLaneAI/pennylane/pull/4564)
 
 * `convert_to_numpy_parameters` now uses `qml.ops.functions.bind_new_parameters`. This reinitializes the operation and
```
Comment (isaacdevlugt) [inline doc/releases/changelog-0.33.0.md:649]: missing PR number

File: doc/releases/changelog-0.33.0.md (hunk 2 above)
Comment (trbromley) [reply doc/releases/changelog-0.33.0.md:300]: I don't think it needs it, we just need to show the principle that we can execute this JITted function

File: doc/releases/changelog-0.33.0.md [hunk 7]
```diff
@@ -240,64 +338,34 @@
   [(#4628)](https://github.com/PennyLaneAI/pennylane/pull/4628)
   [(#4649)](https://github.com/PennyLaneAI/pennylane/pull/4649)
 
+* `DefaultQubit2` can now accept a `jax.random.PRNGKey` as a `seed`, to set the key for the JAX pseudo random
```
[the remark this answers is not in the document]
Comment (trbromley) [reply doc/releases/changelog-0.33.0.md]: Yes, have updated :+1:

File: doc/releases/changelog-0.33.0.md (hunk 6 above)
Comment (trbromley) [reply doc/releases/changelog-0.33.0.md:647]: Fixed

File: doc/releases/changelog-0.33.0.md [hunk 8]
```diff
@@ -61,298 +93,359 @@
...
+  `validate_device_wires`, `validate_multiprocessing_workers`, `warn_about_trainable_observables`,
+  and `no_sampling` to assist in constructing devices under the new device API.
+  [(#4659)](https://github.com/PennyLaneAI/pennylane/pull/4659)
 
-* The density matrix aspects of `StateMP` have been split into their own measurement
-  process, `DensityMatrixMP`.
-  [(#4558)](https://github.com/PennyLaneAI/pennylane/pull/4558)
+* Updated `qml.device`, `devices.preprocessing` and the `tape_expand.set_decomposition` context 
+  manager to bring `DefaultQubit` to feature parity with `default.qubit.legacy` with regards to 
+  using custom decompositions. The `DefaultQubit2` device can now be included in a `set_decomposition`
```
Comment (trbromley) [inline doc/releases/changelog-0.33.0.md:396]:
```suggestion
  using custom decompositions. The `DefaultQubit` device can now be included in a `set_decomposition` 
```

**Review by isaacdevlugt — APPROVED** [review]
🚀

**Pull request merged**